src/app/ipv/middleware.js

```diff
@@ -187,7 +192,31 @@ module.exports = {
   },
   handleJourneyPage: async (req, res, next) => {
     try {
+      const currentEnvironment = process.env.NODE_ENV;
       const { pageId } = req.params;
+
+      if (
+        currentEnvironment === "development" &&
```

Comment:
Might be worth having "development" as a constant or an Enum value to avoid accidental changes in the future?